hopr/hopr-lib/tests/chain_operations.rs

```diff
@@ -246,3 +246,38 @@ async fn test_withdraw_native(#[future(awt)] cluster_fixture: ClusterGuard) -> a
     assert!(final_balance_src < initial_balance_src - withdrawn_amount); // account for gas
     Ok(())
 }
+
+#[rstest]
+#[tokio::test]
+#[serial]
+async fn test_check_ticket_price_is_default(#[future(awt)] cluster_fixture: ClusterGuard) -> anyhow::Result<()> {
+    let [node] = exclusive_indexes::<1>();
+
+    let ticket_price = cluster_fixture[node]
+        .inner()
+        .get_ticket_price()
+        .await
+        .context("failed to get ticket price")?;
+
+    assert!(ticket_price > hopr_lib::HoprBalance::zero());
```

Comment:
This does not, however test, what the test name describes. It tests, whether the ticket price is set to a non-null value. you'd need to fetch the oracle value, then compare to the API.